tests/bugs/invalid-entrypoint-param.slang

```diff
@@ -0,0 +1,17 @@
+//TEST:SIMPLE(filecheck=CHECK): -target spirv
+
+// `TT` is not valid for defining a varying entrypoint parameter,
+// and we should diagnose an error.
+
+// CHECK: error
```

Comment:
This would be better if this check has an explicit error number, 39028, because it could match to an unexpected error elsewhere.